baseframe/__init__.py

```diff
@@ -248,6 +248,14 @@ def init_app(
         app.assets.register('css_all', css_all)
         app.register_blueprint(self, static_subdomain=subdomain)
 
+        for module_name in app.config.get('PRIVATE_ASSETS', []):
```

Comment:
Don't use config as it's a statutory requirement for the client app and not a configuration issue. Take it as a parameter named `asset_modules` instead.